gatsby/src/components/info.js

```diff
@@ -25,10 +25,48 @@ function URLify(string) {
 
 const Info = ({ uri, location, data, yaml, ...rest }) => {
   const { title, language } = useHelmetTags(uri, data[yaml]);
+  const [emailCollectionFormState, setEmailCollectionFormState] = React.useState({});
+  const [emailCollectionFormMessage, setEmailCollectionFormMessage] = React.useState({});
```

Comment:
This may be overkill. Let me know if you want to handle this in a different way or not at all.